cli/cmd/install.go

```diff
@@ -340,6 +361,12 @@ func (options *installOptions) validateAndBuild(stage string, flags *pflag.FlagS
 	values.Identity = identityValues
 	values.stage = stage
 
+	caTrust, err := options.generateCATrust()
```

Comment:
mind moving `values.stage = stage` below the `CATrust` block (or somewhere more prominent, as it affects so much behavior)